backend/app/Chat/Servers/ServerQueue.php

```
@@ -122,6 +122,102 @@ public static function hasAtLeastOnePendingItem(string $user): bool
+
+            return [
+                'memory' => 0,
+                'disk' => 0,
+                'cpu' => 0,
+                'databases' => 0,
+                'backups' => 0,
+                'allocations' => 0,
+                'servers' => 0
+            ];
+        }
+    }
```

PHPMD (2.15.0)

The parameter $user_uuid is not named in camelCase. (undefined)

(CamelCaseParameterName)